c2cgeoportal/views/entry.py

```diff
@@ -244,10 +244,12 @@ def _layer(self, layer, wms, wms_layers, time):
                 self._fill_wmts(l, layer, wms, wms_layers, errors)
         elif isinstance(layer, LayerInternalWMS):
             l["type"] = "internal WMS"
+            l["layers"] = layer.layer
```

Comment:
No, in fact `layer.layer` should be named as `layer.layers` and can be a coma separated list of layers...